app/be-a-genie/page.tsx

```diff
@@ -1,28 +1,56 @@
-"use client";
+'use client';
 import dynamic from 'next/dynamic';
-import Footer from '@/components/GenieComponents/Footer';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { GoogleReCaptchaProvider } from 'react-google-recaptcha-v3';
 import Become from '@/components/GenieComponents/Become';
-import GetReady from '@/components/GenieComponents/GetReady';
-import Hero from '@/components/GenieComponents/Hero';
+import Hero from '@/components/GenieComponents/GenieHero';
+import Footer from '@/components/common/Footer';
+import GetReady from '@/components/common/GetReady';
 
-const GenieBenefits = dynamic(() => import('@/components/GenieComponents/GenieBenefits'), { ssr: false });
+const GenieBenefits = dynamic(() => import('@/components/GenieComponents/GenieBenefits'), {
+  ssr: false,
+});
+const Navbar = dynamic(() => import('@/components/common/Navbar'), { ssr: false });
+
+const genieMenuItems = [
+  { label: 'Become a Genie', sectionId: 'about' },
+  { label: 'Genie Benefits', sectionId: 'features' },
+];
+
+const getReadyContent = {
+  mobile: ['Get Ready for', '$ervgenie Service', 'Magic is Just Around', 'the Corner!'],
```

Comment:
it looks like the contents got flipped. this message should be for the home page, and the on for the genie should be here.
<img width="1174" alt="Screenshot 2024-12-03 at 1 08 13 AM" src="https://github.com/user-attachments/assets/03053623-eb52-4384-beaa-a93cc92c6a72">

Reply:
fixed